content/posts/2018/02/2018-02-05-using-journey-mapping-streamline-processes-across-agencies.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: using-journey-mapping-streamline-processes-across-agencies
+date: 2018-02-05 11:52:00 -0400
+title: 'Using Journey Mapping to Streamline Processes Across Agencies'
+summary: 'Summary of an agency information sharing exercise to improve the customer experience for newly naturalized citizens attempting to travel abroad&#46;'
+authors: 
+  - lgodfrey
+categories: 
+  - content
+  - design
+  - metrics
+  - ux
+tag: 
+  - content
+  - design
```

Comment:
This isn't appearing as a tag because it isn't recorded as a tag in the master tag document: https://github.com/GSA/digitalgov.gov/blob/master/data/tag.yml

Should this be a tag?

Reply:
Oh, not design -- thanks! I'll remove